src/test/functional/tests/buzzPage.spec.ts

```diff
@@ -0,0 +1,28 @@
+// buzzPage.spec.ts
+import { LoginPage } from "../pages/LoginPage";
+import { BuzzPage } from "../pages/BuzzPage";
+import { test, expect } from "@playwright/test";
+import { adminUserTestData } from "../data";
+import { buzzPageAssertions } from "../assertions";
+
+test.describe("Login Page", () => {
+  let loginPage: LoginPage;
+  let buzzPage: BuzzPage;
+
+  test.beforeEach(async ({ page }) => {
+    loginPage = new LoginPage(page);
+    buzzPage = new BuzzPage(page);
+    await loginPage.initialize();
+    await loginPage.navigateToMainPage();
+    await loginPage.loginUser(adminUserTestData.userName, adminUserTestData.password);
+  });
+
+  test("Test Case 1: Share file and Delete file", async ({ page }) => {
```

Comment:
Delete file potraktowalabym jako osobny test case

Reply:
dzieki takiemu podejsciu usuwam odrazu plik i czyszcze sobie dane testowe

Reply:
2 pieczenie na 1 ogniu

Reply:
moge zrobic osobny test w ktorym bede dodawal plik i usuwal go, nie chce robic w test case 1 sharowaniu pliku i test case 2 usuwanie tego pliku bo wtedy testy sa od siebie zalezne, a czy jest sens robic 2 testy w ktorych sharuje pliki i usuwam tylko w 1 nazwa bedzie sharing a w 2 nazwa bedzie deleting a testy beda robic to samo?